editor/find_in_files.cpp

```diff
@@ -705,12 +706,13 @@ void FindInFilesPanel::_notification(int p_what) {
 
 void FindInFilesPanel::_on_result_found(String fpath, int line_number, int begin, int end, String text) {
 	TreeItem *file_item;
+	Ref<Texture2D> remove_texture = get_editor_theme_icon(SNAME("Close"));
 	HashMap<String, TreeItem *>::Iterator E = _file_items.find(fpath);
-
 	if (!E) {
 		file_item = _results_display->create_item();
 		file_item->set_text(0, fpath);
 		file_item->set_metadata(0, fpath);
+		file_item->add_button(0, remove_texture, -1, false, TTR("Remove result"));
```

Comment:
We use Sentence Case for buttons:
```suggestion
		file_item->add_button(0, remove_texture, -1, false, TTR("Remove Result"));
```
Same for other occurrences.

Reply:
Ah wait, if this is a tooltip then it's correct like this with Sentence case.